src/building-blocks/Info.tsx

```diff
@@ -8,25 +8,41 @@ import { ClassValue } from 'classnames/types';
 import * as Antd from 'antd';
 
 import { CLASS_PREFIX } from '../consts';
-import { IFieldConfigPartial } from '../interfaces';
+import { IFieldConfigPartial, IFormatProps } from '../interfaces';
 
 @autoBindMethods
 @observer
-class Info extends Component<{ fieldConfig: IFieldConfigPartial }> {
+class Info extends Component<{ fieldConfig: IFieldConfigPartial; format?: IFormatProps }> {
   public render() {
+    const { format } = this.props,
+      layout = format && format.layout,
+      rowClassName = layout !== 'inline' ? '' : `${CLASS_PREFIX}-info-item-inline`;
+
     return (
       <Antd.Col {...this.props.fieldConfig.colProps} className={`${CLASS_PREFIX}-info`}>
-        {this.props.children}
+        <Antd.Row className={rowClassName}>{this.props.children}</Antd.Row>
       </Antd.Col>
     );
   }
 }
 
 @autoBindMethods
 @observer
-class Label extends Component<{ className?: ClassValue }> {
+class Label extends Component<{ className?: ClassValue; format?: IFormatProps }> {
   public render() {
-    return <div className={cx(this.props.className, `${CLASS_PREFIX}-info-label`)}>{this.props.children}</div>;
+    const { className, format } = this.props,
+      colon = format && format.colon,
+      layout = format && format.layout,
+      // colon can be displayed only when layout is inline or horizontal
+      colonClassName =
+        colon === false || layout === 'vertical' ? `${CLASS_PREFIX}-info-label-no-colon` : `${CLASS_PREFIX}-info-label`,
```

Comment:
Since  `${CLASS_PREFIX}-info-label` is used so much, I wonder if we should keep it, and add a class for colon and no-colon. And hell, we could add layout as well.

`cx(
  `${CLASS_PREFIX}-info-label`,
  `${CLASS_PREFIX}-info-label${colon ? '' : '-no'}-colon`
  `${CLASS_PREFIX}-info-label-layout-${layout}`
)`